journal/modernized/42-appendix.adoc

```diff
@@ -2173,6 +2178,7 @@ articles that has been brought to New Zealand:
 it is astonishingly great.
 I remember to have heard a missionary state,
 that in one pa or native fort,--of which he was speaking,
+//pa above is circled, but it matches the original. Not sure what it means.
```

Comment:
Ok, if it matches the original, then its probably just an island term. We can leave it.